en/ros/ros2_comm.md

```diff
@@ -1,340 +1,211 @@
-# PX4-ROS 2 Bridge
+# ROS 2 User Guide (PX4-ROS 2 Bridge)
 
-This section of the guide focusses on the bridge between PX4 and ROS 2, made available through the [microRTPS Bridge](../middleware/micrortps.md) and resulting products to interface with ROS 2, *px4_ros_com* and *px4_msgs* ROS 2 packages.
+This topic explains how to setup and use ROS 2 with PX4.
 
-It also provides information in how to:
-1. Connect ROS 2 nodes with PX4 (via the *microRTPS* Bridge, and using the `px4_ros_com` package)
-1. Connect ROS (ROS "version 1") nodes with PX4 by additionally using the `ros1_bridge` package to bridge ROS 2 and ROS (1).
+It provides an overview of the ROS2-PX4 bridge architecture and application pipeline, along with instructions on how to install all the needed software and build ROS 2 applications.
 
-:::note
-For information about using the *PX4-Fast RTPS(DDS) Bridge* without ROS 2, see the [RTPS/DDS Interface section](../middleware/micrortps.md).
-:::
 
 :::note
-For a more detailed and visual explanation of using the bridge with ROS 2, see the following talks/presentation videos:
-1. [PX4 Dev Summit 2019 - "ROS 2 Powered PX4"](https://www.youtube.com/watch?v=2Szw8Pk3Z0Q)
-1. [ROS World 2020 - Getting started with ROS 2 and PX4](https://www.youtube.com/watch?v=qhLATrkA_Gw)
-:::
-
-## Why Two Bridges (PX4/ROS2 vs PX4/FastDDS)?
```

Comment:
FYI duplication. I no longer explain that IDL files are different than what comes out of microRTPS by default. No one cares :-)

Reply:
Though I guess at some point we might have to have "exposing your own uorb message" topic

Reply:
@hamishwillee yes they do. Because then someone might ask why the agent generated from the PX4 build doesn't work with ROS 2.

Reply:
Maybe. I thought that was important when we'd mixed up all the instructions between getting stuff from px4_msg and the agent. Now we're being a lot more clear that you don't generate your own agent code.

I added a note at the end of the overview anyway.